libp2p/CMakeLists.txt

```diff
@@ -18,6 +18,9 @@ include_directories(${LEVELDB_INCLUDE_DIRS})
 if (MINIUPNPC_FOUND)
 	include_directories(${MINIUPNPC_INCLUDE_DIRS})
 endif()
+
+include_directories(${Boost_INCLUDE_DIRS})
```

Comment:
DLLs were not getting copied to /bin.  MSYS typically has static .a and dll.a link libraries in /lib, headers in /include, and executables and DLLs in /bin. Currently none of the DLLS are getting copied upon install.